WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/WooShippingSplitShipmentsDetailView.swift

```diff
@@ -25,6 +25,23 @@ struct WooShippingSplitShipmentsDetailView: View {
 
                 ScrollView {
                     VStack(alignment: .leading, spacing: Layout.contentPadding) {
+                        if viewModel.currentShipment.isPurchased {
+                            VStack {
+                                Text(Localization.PurchasedShipment.title)
+                                    .bold()
+                                    .frame(maxWidth: .infinity, alignment: .leading)
+                                Text(Localization.PurchasedShipment.subtitle)
+                                    .frame(maxWidth: .infinity, alignment: .leading)
+                            }
+                            .multilineTextAlignment(.leading)
+                            .foregroundStyle(Layout.green)
+                            .padding(Layout.contentPadding)
+                            .background(
+                                Layout.greenBackground
+                                    .clipShape(RoundedRectangle(cornerRadius: Layout.cornerRadius))
+                            )
+                        }
```

Comment:
super-nit: Consider moving this into a computed property to keep the main `body` less crowded.  Perhaps named as `shipmentPurchasedMessageView`.